cnf-certification-test/certification/suite.go

```diff
@@ -197,3 +178,25 @@ func testHelmCertified(env *provider.TestEnvironment) {
 		}
 	})
 }
+
+func isReleaseCertified(helm *release.Release, ourKubeVersion string, out api.ChartStruct) bool {
```

Comment:
move this function into certtools/certtools.go

Reply:
Done.